src/shibuya.js

```diff
@@ -0,0 +1,96 @@
+import { Keyring } from "@polkadot/api";
+import { u8aToHex } from "@polkadot/util";
+import { decodeAddress } from '@polkadot/util-crypto';
+
+import { env, chainConfig } from "./common/constants";
+const { TURING_PARA_ID, SHIBUYA_ENDPOINT, TURING_ENDPOINT, SHIBUYA_PARA_ID } = env;
+
+import shibuyaHelper from "./common/shibuyaHelper";
+import turingHelper from "./common/turingHelper";
+import { sendExtrinsic } from "./common/utils";
+
+const getProxyAccount = (api, address) => {
```

Comment:
this is the getProxyAccount for Shibuya, so please move this function to shibuyaHelper. Let’s keep main() the only function in this file.